lib/acid/strutil.lua

```diff
@@ -248,6 +248,52 @@ function _M.endswith(str, suffix)
 end
 
 
+local function _contains(str, sub_str)
+    local str_hex = _M.tohex(str)
+    local sub_str_hex = _M.tohex(sub_str)
+
+    local mathed = string.match(str_hex, sub_str_hex)
```

Comment:
hex后每个字节变成2字符, 会出现在2n+1的位置匹配上吧?

我觉得..这个是你要的不?

```
string.find (s, pattern [, init [, plain]])
Looks for the first match of pattern in the string s. If it finds a match, then find returns the indices of s where this occurrence starts and ends; otherwise, it returns nil. A third, optional numerical argument init specifies where to start the search; its default value is 1 and can be negative. A value of true as a fourth, optional argument plain turns off the pattern matching facilities, so the function does a plain "find substring" operation, with no characters in pattern being considered "magic". Note that if plain is given, then init must be given as well.
```

Reply:
对，用find应该简单一些，我改下